.github/actions/setup/action.yml

```diff
@@ -29,6 +29,10 @@ runs:
           **/node_modules/
           !node_modules/.cache/nx/
         key: npm-${{ hashFiles('package-lock.json') }}-${{ hashFiles('utils/atomic-storybook/package-lock.json') }}
+    - uses: actions/cache@0c45773b623bea8c8e75f6c82b208c3cf94ea4f9 # v4
+      with:
+        path: packages/quantic/force-app/main/default/staticresources
+        key: quantic-${{ hashFiles('package-lock.json') }}-${{ hashFiles('utils/atomic-storybook/package-lock.json') }}
```

Comment:
Why this key exactly? does it matter?

Reply:
not really, it's just an identifier of that cache, like documented here: https://github.com/actions/cache?tab=readme-ov-file#inputs

From the documentation, using the hashFiles function allows us to create a new cache when dependencies change which is exactly what we want.